src/app/src/main/kotlin/org/vaccineimpact/api/app/errors/InvalidOperationError.kt

```diff
@@ -0,0 +1,7 @@
+package org.vaccineimpact.api.app.errors
+
+import org.vaccineimpact.api.models.ErrorInfo
+
+class InvalidOperationError(message: String) : MontaguError(400, listOf(
+        ErrorInfo("invalid-operation", message)
+))
```

Comment:
Previously we had `OperationNotAllowedError`. This was only being used for impossible operations (e.g. state changes that can't happen), but somewhere along the way confusion has crept in: We were using status 401 Unauthorized, taking a different meaning of "not allowed".

So to clear things up I've renamed it, changed the error code to match, and changed the status to 400. There doesn't seem to be a more specific status code for "illegal operation".

The only user who might be affected is Rich. I'll liaise with him.